src/EditorFeatures/Core.Wpf/Suggestions/SuggestedActionSource_ISuggestedActionsSourceExperimental.cs

```diff
@@ -111,10 +111,8 @@ await foreach (var set in lowPriSet)
                 }
                 else
                 {
-                    await Task.WhenAll(fixesTask, refactoringsTask).ConfigureAwait(false);
-                    var fixes = await fixesTask.ConfigureAwait(false);
-                    var refactorings = await refactoringsTask.ConfigureAwait(false);
-                    foreach (var set in ConvertToSuggestedActionSets(state, selection, fixes, refactorings))
+                    var actionsArray = await Task.WhenAll(fixesTask, refactoringsTask).ConfigureAwait(false);
+                    foreach (var set in ConvertToSuggestedActionSets(state, selection, fixes: actionsArray[0], refactorings: actionsArray[1]))
                         yield return set;
```

Comment:
Back in the olden days we always required a block statement under control flow structures. Did this change or get relaxed?

Reply:
this is relaxed.  the rule is: if it's a single-line you don't need the block.

Reply:
Not going to argue, but in my opinion this is a regression that makes it worse. A few times we've been burned badly after a bad merge or just someone not paying attention. Much harder to make this mistake when curlies are present.

Of course I understand you're not going to change it back now :) Grumble grumble

Reply:
> Much harder to make this mistake when curlies are present.

it should be very hard to make a mistake here.  For example, teh following will not compile:

```c#
if (foo)
    bar();
    bar();
```

This will fail as the formatting will be incorrect.  

--

Note: we haven't had a single issue with instituting this as we also have hte formatting checker.

Reply:
I've mostly learned, refined and solidified my coding style from you during the early Roslyn days, and our StyleCop choices in the old repo. If you look at the code I wrote in the past 10 years it'll be factored and formatted the way you'd do it.

So if you tell me to change it now, I guess I have no choice but adapt! Growth mindset and all.

Reply:
:)

i was resistent to this at first.  but we wanted to adopt the .net coding pattern, so this came along.  I found it actually somewaht reasonable.  The one thing i can't abide by is:

```c#
if (foo)
   bar();
else
   baz();
```

That just still looks awful to me :)

Importantly though, the block can be avoided *only* if the body is on a *single line*.  so this is allowed:

```c#
if (foo)
   bar(baz(), quux());
```

But this is not:

```c#
if (foo)
    bar(
        baz(), quux());
```

The restrictions make this look good, while avoiding lots of confusing cases.

Reply:
When I see formatting rules like this I just close my eyes and think of my side projects 😛